docs/release-notes/8.4.6-6.md

```diff
@@ -0,0 +1,56 @@
+# Percona XtraDB Cluster 8.4.6-6 (2025-09-)
+
+Percona XtraDB Cluster (PXC) supports critical business applications in both cloud and on-premises environments, including public, private, and hybrid setups. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+## Release highlights
+
+Percona XtraDB Cluster 8.4.6-6 is based on Percona Server for MySQL and includes all improvements and bug fixes available in the [Percona Server for MySQL 8.4.6-6 (2025-09-08) release notes](https://docs.percona.com/percona-server/8.4/release-notes/8.4.6-6.html).
+
+### Percona XtraDB Cluster 8.4.6-6
+
+Improves State Snapshot Transfer (SST) by retrying Incremental State Transfer (IST) when initial SST attempts fail before any data changes, preventing unnecessary full data transfers.
+
+### Percona Server for MySQL 8.4.6-6
+
+Implements Link-Time Optimization (LTO) to generate more optimized release binaries, resulting in improved performance for specific workloads.
+
+### MySQL 8.4.6
+
+Bug fixes introduced by Oracle for MySQL 8.4.6 and included in Percona Server for MySQL are the following:
+
+* Fixed an issue where rebuilding a primary key with duplicate entries could cause the server to stop unexpectedly. (Bug #37822992)
+
+* Fixed an issue related to dropping columns that were part of an index. (Bug #37726881)
+
+* Fixed an issue with indexing spatial datatype columns. (Bug #36682518)
+
+* Fixed an issue where creating a secondary index on a `VARCHAR` column could exceed configured memory limits, with the amount allocated being directly related to the `value of innodb_ddl_buffer_size`, leading to errors such as ERROR 1136 (21S01): Column count doesn't match value count at row 1. (Bug #37233273)
+
+Find the complete list of bug fixes and changes in the [MySQL 8.4.6 release notes](https://dev.mysql.com/doc/relnotes/mysql/8.4/en/news-8-4-6.html).
+
+## New features
+
+* [PXC-4631](https://perconadev.atlassian.net/browse/PXC-4631): Improves State Snapshot Transfer (SST) by retrying Incremental State Transfer (IST) when initial SST attempts fail before any data changes, preventing unnecessary full data transfers.
+
+## Bug fixes
+
+* [PXC-4392](https://perconadev.atlassian.net/browse/PXC-4392): The default timeout logic for State Snapshot Transfer (SST) was insufficient and resulted in SST failures.
+
+* [PXC-4644](https://perconadev.atlassian.net/browse/PXC-4644): Fixed an issue where adding a foreign key (FK) without sufficient privileges caused the executing node to fail with error 1216, and the replicating nodes to fail with error 1452.
+
+## Build & packaging notes
+
+* The official packages were built with the `WITH_LTO=ON` flag to enable the Link-Time Optimization (LTO) feature.
+
+* Percona Server for MySQL 8.4.6.6 suports Red Hat Enterprise Linux (RHEL) 10.
```

Comment:
This release adds support for Red Hat Enterprise Linux 10.

This is a PXC release - the PS reference is out of place.